src/InfoViz/Native/ParallelCoordinates/index.js

```diff
@@ -294,6 +297,7 @@ function parallelCoordinate(publicAPI, model) {
           .attr('width', glyphSize)
           .attr('height', glyphSize)
           .style('color', d.legend.color) // Firefox SVG use color bug workaround fix
+          .classed(style.clickable, (data, idx) => d.annotated)
```

Comment:
I don't think this is doing what you want? (data) as an arg, (d.annotated) in the body?